repose-aggregator/components/services/http-client-service/http-client-service-api/src/main/resources/META-INF/schema/examples/http-connection-pool.cfg.xml

```diff
@@ -25,4 +25,13 @@
     <!-- Configuration for a custom pool that uses all default values -->
     <pool id="alldefaultspool" default="false"/>
 
+    <!-- Configuration for a connection pool that uses SSL/TLS Client Authentication -->
+    <pool id="clientAuthentication"
+          default="false"
+          keystore-filename="keystore.jks"
+          keystore-password="password"
+          key-password="secret"
+          truststore-filename="truststore.jks"
```

Comment:
The `truststore` is not a requirement, but if we are setting this up we might as well do it now before someone asks.